erpnext/accounts/doctype/pos_invoice/pos_invoice.py

```diff
@@ -475,9 +477,12 @@ def get_stock_availability(item_code, warehouse):
 		bin_qty = get_bin_qty(item_code, warehouse)
 		pos_sales_qty = get_pos_reserved_qty(item_code, warehouse)
 		return bin_qty - pos_sales_qty
-	else:
-		if frappe.db.exists('Product Bundle', item_code):
-			return get_bundle_availability(item_code, warehouse)
+	elif frappe.db.exists('Product Bundle', item_code):
+		return get_bundle_availability(item_code, warehouse)
+	#To continue the flow considering a service item
+	elif frappe.db.get_value('Item', item_code, 'is_stock_item') == 0:
+		return 0
+
```

Comment:
```suggestion

	else:
		if frappe.db.exists('Product Bundle', item_code):
			return get_bundle_availability(item_code, warehouse)
		else:
			# Is a service item
			return 0
```

Reply:
Also, we need to show service items as available without quantity in the Item Details section